shell/common/vsync_waiter_fallback.cc

```diff
@@ -2,9 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#define FML_USED_ON_EMBEDDER
```

Comment:
Since this is not in the embedder, it seems odd to lift this gate here. This was done so that custom embedders with no custom event loop integration would continue to function. I think this may not be necessary anymore though.

Anyway, the base `fml::TaskRunner` has a protected `task_runners_` field from you can grab a reference to the task runner you need. No need to use this define to access the current task runner.

Reply:
Done!